### PR TITLE
add README to KHI test and include it in documentation 

### PR DESCRIPTION
**Pull request by PrometheusPi**
The KHI test was missing a README file. This pull request adds a short description? 
@psychocoderHPC should I also add a quick example on how to run it?

### CODE REVIEW

File: share/picongpu/tests/KHI_growthRate/README.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+KelvinHelmholtz: Kelvin-Helmholtz Instability
+=============================================
+
+.. sectionauthor:: Richard Pausch <r.pausch (at) hzdr.de>
+.. moduleauthor:: Richard Pausch <r.pausch (at) hzdr.de>, Mika Soren Voß <m.voss (at) hzdr.de>
+
+This KHI growth rate test simulates a shear-flow instability known as the Kelvin-Helmholtz Instability in a sub-relativistic setup as studied in [Alves12]_, [Grismayer13]_, [Bussmann13]_. The setup uses a pre-ionized quasi-neutral hydrogen plasma. From the simulation, the growth of the dominant magnetic field is extracted and compared to theoretic predictions. The tests only passes if the PIConGPU reaches the ideal theoretical growth rate with a 10% accuracy.
```
Comment (steindev) [inline share/picongpu/tests/KHI_growthRate/README.rst]: 'the PIConGPU'?

File: share/picongpu/tests/KHI_growthRate/README.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
+KelvinHelmholtz: Kelvin-Helmholtz Instability
+=============================================
+
+.. sectionauthor:: Richard Pausch <r.pausch (at) hzdr.de>
+.. moduleauthor:: Richard Pausch <r.pausch (at) hzdr.de>, Mika Soren Voß <m.voss (at) hzdr.de>
+
+This KHI growth rate test simulates a shear-flow instability known as the Kelvin-Helmholtz Instability in a sub-relativistic setup as studied in [Alves12]_, [Grismayer13]_, [Bussmann13]_. The setup uses a pre-ionized quasi-neutral hydrogen plasma. From the simulation, the growth of the dominant magnetic field is extracted and compared to theoretic predictions. The tests only passes if the PIConGPU reaches the ideal theoretical growth rate with a 10% accuracy. 
+		  
+
```
Comment (steindev) [inline share/picongpu/tests/KHI_growthRate/README.rst:20]: A 'How to Execute' section would be very helpful.

**Review by steindev — CHANGES_REQUESTED** [review]
Please see comments.

**Comment by steindev** [comment]
And is this text supposed to appear in the readthedocs? By now it is not included.

**Comment by psychocoderHPC** [comment]
@PrometheusPi Could you please finish this PR.

File: share/picongpu/tests/KHI_growthRate/README.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
+KelvinHelmholtz: Kelvin-Helmholtz Instability
+=============================================
+
+.. sectionauthor:: Richard Pausch <r.pausch (at) hzdr.de>
+.. moduleauthor:: Richard Pausch <r.pausch (at) hzdr.de>, Mika Soren Voß <m.voss (at) hzdr.de>
+
+This KHI growth rate test simulates a shear-flow instability known as the Kelvin-Helmholtz Instability in a sub-relativistic setup as studied in [Alves12]_, [Grismayer13]_, [Bussmann13]_. The setup uses a pre-ionized quasi-neutral hydrogen plasma. From the simulation, the growth of the dominant magnetic field is extracted and compared to theoretic predictions. The tests only passes if the simulation reaches the ideal theoretical growth rate with a 10% accuracy.
```
Comment (steindev) [inline share/picongpu/tests/KHI_growthRate/README.rst]: theroretic -> theoretical
The tests only passes ... -> The tests only pass ...

File: share/picongpu/tests/KHI_growthRate/README.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+.. [Alves12]
+       E.P. Alves, T. Grismayer, S.F. Martins, F. Fiuza, R.A. Fonseca, L.O. Silva.
+       *Large-scale magnetic field generation via the kinetic kelvin-helmholtz instability in unmagnetized scenarios*,
+       The Astrophysical Journal Letters (2012),
+       https://dx.doi.org/10.1088/2041-8205/746/2/L14
+
+.. [Grismayer13]
+       T. Grismayer, E.P. Alves, R.A. Fonseca, L.O. Silva.
+       *dc-magnetic-field generation in unmagnetized shear flows*,
```
Comment (steindev) [inline share/picongpu/tests/KHI_growthRate/README.rst]: Phys Rev writes the title as 'dc-Magnetic-Field Generation in Unmagnetized Shear Flows'.

**Review by steindev — CHANGES_REQUESTED** [review]
Some typos remain. Please squash all commits before next push.

**Comment by PrometheusPi** [comment]
@steindev I added all your suggestions. And I added a comment on the need of `$PICSRC` which we found out today with @psychocoderHPC.

**Review by chillenzer — APPROVED** [review]
In the hope of steering this into the direction of a merge: Looks good. Title of the PR is a bit misleading as you added a full section to the docs. Maybe you want to change this, so our [automatic changelog](https://github.com/ComputationalRadiationPhysics/picongpu/pull/4908) is more accurate to start with.

**Comment by PrometheusPi** [comment]
ping @psychocoderHPC and @steindev: ready for merge

**Review by steindev — APPROVED** [review]

**Pull request merged**